site/_posts/2020-09-10-building-profile-portal-with-flask-oauth-apis.md

```diff
@@ -67,36 +83,47 @@ These changes ensure that after the user signs in to FusionAuth, they can be sen
 
 {% include _image.liquid src="/assets/img/blogs/flask-oauth-portal/oauth-tab-of-application.png" alt="Configuring the FusionAuth application for the flask portal." class="img-fluid" figure=false %}
 
+## Setting up FusionAuth using a Kickstart file
+
+Instead of manually setting up FusionAuth above, you can also use Kickstart. This lets you get going quickly if you have a fresh installation of FusionAuth. Learn more about how to use Kickstart here [https://fusionauth.io/docs/v1/tech/installation-guide/kickstart/](https://fusionauth.io/docs/v1/tech/installation-guide/kickstart/). Here's an example Kickstart file [https://github.com/FusionAuth/fusionauth-example-kickstart/blob/matt/kickstart-python/example-apps/python-flask-registration-forms.json](https://github.com/FusionAuth/fusionauth-example-kickstart/blob/master/example-apps/python-flask-registration-forms.json) which sets up FusionAuth for this tutorial.
```

Comment:
can you change the links to have English words as the anchor text, rather than the urls. (Same as we did for the other post).

Also, the link to the JSON file is to your branch, please update it for the master branch.